frontend/src/pages/samples/config.tsx

```diff
@@ -518,44 +530,60 @@ export const wesSampleColDefs: Array<ColDef<DashboardSample>> = [
   {
     field: "bamCompleteDate",
     headerName: "Latest BAM Complete Date",
+    headerTooltip: "Most recent date the BAM generation tasks were run",
+    headerComponentParams: createCustomHeader(lockIcon + toolTipIcon),
     valueFormatter: (params) => formatCellDate(params.value) ?? "",
     ...getAgGridDateColFilterConfigs(),
+    width: 270,
   },
   {
     field: "bamCompleteStatus",
     headerName: "BAM Complete Status",
+    headerTooltip: "Indicates whether BAM generation was successful",
+    headerComponentParams: createCustomHeader(lockIcon + toolTipIcon),
+    width: 240,
   },
   {
     field: "mafCompleteDate",
     headerName: "Latest MAF Complete Date",
+    headerTooltip:
+      "Most recent date the MAF generation tasks were run. Valid for tumor samples only",
+    headerComponentParams: createCustomHeader(lockIcon + toolTipIcon),
     valueFormatter: (params) => formatCellDate(params.value) ?? "",
     ...getAgGridDateColFilterConfigs(),
+    width: 270,
   },
   {
     field: "mafCompleteNormalPrimaryId",
     headerName: "MAF Complete Normal Primary ID",
+    headerTooltip:
+      "Primary ID of Normal sample that tumor was paired with, for MAF generation",
+    headerComponentParams: createCustomHeader(lockIcon + toolTipIcon),
+    width: 310,
   },
   {
     field: "mafCompleteStatus",
-    headerName: "MAF Complete Status",
-  },
-  {
-    field: "qcCompleteDate",
-    headerName: "Latest QC Complete Date",
-    valueFormatter: (params) => formatCellDate(params.value) ?? "",
-    ...getAgGridDateColFilterConfigs(),
+    headerName: "MAF Complete Status/Data Eligible for Sharing",
+    headerTooltip:
+      "Indicates whether MAF generation was successful. Valid for tumor samples only. For the MSK WES Repository cohort we only included samples with a MAF Complete Status of 'Pass' as well as QC Complete Result of 'Pass' or 'Warn'",
+    headerComponentParams: createCustomHeader(lockIcon + toolTipIcon),
+    width: 400,
```

Comment:
Can we shrink the width of the tooltip box down? It's very wide.

Reply:
Oh - Is it wide because of the new header name? Can we split the headerName to two lines and put "Data Eligible for Sharing" in parentheses in the second line?
```
MAF Complete Status
(Data Eligible for Sharing)
```